src/api/mutations/account/useEditAccountMutation.ts

```diff
@@ -1,23 +1,24 @@
 import { useMutation, useQueryClient } from '@tanstack/react-query';
 import toast from 'react-hot-toast';
+import { useTranslation } from 'react-i18next';
 
 import httpClient from '@src/api/httpClient';
 import { ENDPOINTS } from '@src/config/endpoints.config';
 import { QUERY_KEYS } from '@src/config/react-query.config';
-import { ProfileActions } from '@src/types/i18n.types';
+import { ToastMessages } from '@src/types/i18n.types';
 import { Profile } from '@src/types/profile.types';
-import { t } from 'i18next';
 
 function useEditAccountMutation(profileId: string) {
   const queryClient = useQueryClient();
+  const { t } = useTranslation();
   return useMutation({
     mutationKey: [QUERY_KEYS.mutation.editAccount],
     mutationFn: async (propertyToChange: Partial<Profile>) => {
       await httpClient.patch(ENDPOINTS.account.updateUserProfile(profileId), propertyToChange);
     },
     onSuccess: async () => {
       await queryClient.invalidateQueries({ queryKey: [QUERY_KEYS.query.user] });
-      toast.success(t(ProfileActions.profile_change));
+      toast.success(`${t(ToastMessages.SuccessUpdated)}`);
```

Comment:
What was successfully updated? 

It is not understandable looking at the message name. Please, don't shorten message names

Reply:
Check all changes for this issue, especially utils messages, names should reflect the message